backend/src/services/secret-import/secret-import-dal.ts

```diff
@@ -70,6 +97,28 @@ export const secretImportDALFactory = (db: TDbClient) => {
     }
   };
 
+  const getProjectImportCount = async (
+    { search, ...filter }: Partial<TSecretImports & { projectId: string; search?: string }>,
+    tx?: Knex
+  ) => {
+    try {
+      const docs = await (tx || db.replicaNode())(TableName.SecretImport)
+        .where(filter)
+        .where("isReplication", false)
```

Comment:
question: what about the imports that replicate? Those are imports too

Reply:
@sheensantoscapadngan noticed imports that replicate count twice for a single import in pagination, so when getting the count getting just the non-replicas provides the correct amount